packages/react-material-ui/src/components/Drawer/DrawerItem.tsx

```diff
@@ -9,10 +10,24 @@ export type DrawerItemProps = {
   active?: boolean;
   collapsed?: boolean;
   onClick?: () => void;
+  fontFamily?: TypographyProps['fontFamily'];
+  fontSize?: TypographyProps['fontSize'];
+  fontWeight?: TypographyProps['fontWeight'];
+  color?: TypographyProps['color'];
```

Comment:
To be organized, this can be something like textProps, so we will know that the styles will be for the text, and the code above will be cleaner

Reply:
This comment applies to all places where you are doing this...